### DISCUSSION TITLE
Posts and tags display locally, disappear when deployed. Site otherwise fine.

### DISCUSSION THREAD

**Discussion by colinspear** [adityatelange/hugo-PaperMod#929] [Q&A]
I am hosting on AWS Amplify and everything renders wonderfully except the list of posts on the home page and the list of tags on the tags page, even though they show up as expected when I run the site locally. I've tried the following with no luck:

1. Host elsewhere
2. Change `mainSections` parameter as explained in [this](https://github.com/adityatelange/hugo-PaperMod/discussions/724) post
3. Switched between Regular Mode and Home info Mode
4. Pored through the source (though I am admittedly not well versed in Hugo templating).

It's doubly confusing because the `index.html` in the `public` folder in my site's repo shows the posts. Also every tag has a folder and when I type the url of a specific tag, the posts will render as expected. 

I'm really at a loss and if anybody has any ideas, please let me know! I've put links to both my website and it's repo below as well as screenshots of the issue.

Also, I'm new to this github discussion thing, so please let me know if I'm using the wrong etiquette. Thanks!

site: https://www.colinspear.com
repo: https://github.com/colinspear/colinspear-site

Deployed home page missing posts:
<img width="1552" alt="deploy-home-page" src="https://user-images.githubusercontent.com/10899864/170593605-ef1bf79a-a817-4251-ab85-5518d2cf7026.png">

Local home page with posts:
<img width="1552" alt="local-home-page" src="https://user-images.githubusercontent.com/10899864/170593619-715f798a-6af6-46ec-a743-dc2929d253e7.png">

Deployed tags page missing posts:
<img width="1552" alt="deploy-tags" src="https://user-images.githubusercontent.com/10899864/170593613-93c9716f-ec28-4c73-9693-dff4a7972a23.png">

Local tags page with posts:
<img width="1552" alt="local-tags" src="https://user-images.githubusercontent.com/10899864/170593623-84479f15-2829-4f9c-a70f-b70ce4825cf9.png">

Deployed page for `links` tag rendering correctly:
<img width="1552" alt="deploy-tags-specific" src="https://user-images.githubusercontent.com/10899864/170593617-2ad6ac25-d877-4a57-810f-c88c959d0142.png">

**Comment by adityatelange** [comment]
Content dir is a private repo as a submodule.
Configure aws to fetch these.

**Comment by colinspear** [comment]
Thanks for putting me on the right track. I've found some resources about how to deal with this and will report back once I've successfully configured AWS. Cheers

**Comment by colinspear** [comment]
It worked! Thank you so much (and thanks for the great theme). Now I know to check the build logs even if the build is successful.

**Comment by adtygan** [comment]
I'm pretty much facing the exact issue. Could you post resources you used to solve this? I'm using github pages to host my website but I think the process should be similar.

**Reply by colinspear** [reply]
It's been so long, I'm afraid I've forgotten my exact steps. I can try to go back and figure it out, but I would recommend googling the key words in @adityatelange first response to my initial issue (something like "configure aws amplify to fetch private repo as a submodule").

**Reply by adtygan** [reply]
Sorry for my late reply. It slipped through my notifications. I was able to get it running. Thank you for getting back though.
